src/test/kotlin/no/nav/familie/oppdrag/grensesnittavstemming/AvstemmingMapperTest.kt

```diff
@@ -0,0 +1,123 @@
+package no.nav.familie.oppdrag.grensesnittavstemming
+
+import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsoppdrag
+import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsperiode
+import no.nav.familie.oppdrag.iverksetting.OppdragMapper
+import no.nav.familie.oppdrag.repository.OppdragProtokoll
+import no.nav.virksomhet.tjenester.avstemming.meldinger.v1.*
+import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import kotlin.test.assertEquals
+
+class AvstemmingMapperTest {
```

Comment:
_Jeg_ skulle gjerne sett flere tester av ekstremverdier og nullverdier. Men kan være gode grunner til at det ikke vil oppstå, og er bortkastet tid? Mappingen består av mange "delmappinger" - kanskje dele opp i testklasser for hver av disse?